Hymns/Home/HomeContainerView.swift

```diff
@@ -7,49 +7,38 @@ struct HomeContainerView: View {
 
     var body: some View {
         NavigationView {
-            TabView(selection: $selectedTab) {
-                HomeView(viewModel: Resolver.resolve())
-                    .tabItem {HomeTab.home.getImage(selectedTab == HomeTab.home)}
-                    .accessibility(label: HomeTab.home.label)
-                    .tag(HomeTab.home)
-                    .hideNavigationBar()
-
-                BrowseView()
-                    .tabItem { HomeTab.browse.getImage(selectedTab == HomeTab.browse)}
-                    .accessibility(label: HomeTab.browse.label)
-                    .tag(HomeTab.browse)
-                    .hideNavigationBar()
-
-                FavoritesView()
-                    .tabItem {HomeTab.favorites.getImage(selectedTab == HomeTab.favorites)}
-                    .accessibility(label: HomeTab.favorites.label)
-                    .tag(HomeTab.favorites)
-                    .hideNavigationBar()
-
-                SettingsView()
-                    .tabItem {HomeTab.settings.getImage(selectedTab == HomeTab.settings)}
-                    .accessibility(label: HomeTab.settings.label)
-                    .tag(HomeTab.settings)
-                    .hideNavigationBar()
-            }.onAppear {
-                // Make the unselected tabs black insetad of grey.
-                UITabBar.appearance().unselectedItemTintColor = .black
-            }.edgesIgnoringSafeArea(.top)
+            IndicatorTabView(currentTab: $selectedTab, tabItems: [
+                .home,
+                .browse,
+                .favorites,
+                .settings
+                ]).hideNavigationBar()
+            .edgesIgnoringSafeArea(.top)
```

Comment:
I don't think we should ignore safe area top. To me it looks better with the space.